Makefile

```diff
@@ -26,7 +26,7 @@ build-cli: clean
 	cat ./cli/build/sha256sum.txt
 
 build:
-	docker build --build-arg VERSION=$(VERSION) . -t infoblox/migrate -t infoblox/migrate:$(VERSION)
+	docker build --pull --build-arg VERSION=$(VERSION) . -t infoblox/migrate -t infoblox/migrate:$(VERSION)
```

Comment:
Not sure why, but this was added in same PR #21.
`--pull` option allows docker to use latest version of base images instead of locally stored version.
Not sure if I should change it in this PR.
This branch has `ib` branch as base and has it's changes.